be/src/exec/pipeline/aggregate/aggregate_blocking_sink_operator.cpp

```diff
@@ -12,12 +12,12 @@ Status AggregateBlockingSinkOperator::prepare(RuntimeState* state) {
     return _aggregator->open(state);
 }
 
-void AggregateBlockingSinkOperator::close(RuntimeState* state) {
+Status AggregateBlockingSinkOperator::close(RuntimeState* state) {
     _aggregator->unref(state);
-    Operator::close(state);
+    return Operator::close(state);
```

Comment:
not suggest add Status to close.
if we write such code. may be cause a leak.
close method should be exception safe.
```
RETURN_IF_ERROR(child->close());
RETURN_IF_ERROR(this->close());
```

Reply:
Operator::close(state) is always the last statement of its derived classes' close method